site/src/pages/release-notes/0.99.7.mdx

```diff
@@ -0,0 +1,246 @@
+  clientWithoutBaseUrl.get("https://foo.com/");
+  // Bad
+  clientWithBaseUrl.get("https://foo.com/");
+  clientWithoutBaseUrl.get("/bar");
+  ```
+- Boolean parameter conversion became more strict in annotated services. #2767 #2774
+  - Only `true`, `false`, `1` and `0` are accepted. Other values will cause a `400 Bad Request` response.
+- <type://DocService> web UI now shows the 'request body' field for `DELETE` and `PATCH` methods in the debug
+  form. #2756 #2819
+- <type://JacksonRequestConverterFunction> now handles the case where the target type has a type parameter,
+  e.g. `List<Long>`. #2769 #2779
+- Fixed a bug where the current <type://ServiceRequestContext> is not pushed when invoking
+  <type://ResponseConverterFunction#convertResponse(ServiceRequestContext,ResponseHeaders,Object,HttpHeaders)>.
+  #2789
+- <type://RequestContextExportingAppender> now handles the `<exports />` tag correctly. #2781
+- Fixed a bug where some Reactive Streams `Subscriber` implementations violate the specification. #2815
+- You no longer get sporadic <type://WriteTimeoutException> from proxied connections. #2801 #2805
+- You no longer get a <type://CancelledSubscriptionException> unnecessarily when using
+  <type://PublisherBasedStreamMessage>. #2797
+- You no longer get sporadic `EncoderException`s from HTTP/1 connections. #2765
+- It's now disallowed to specify the following headers in gRPC `Metadata`. #2718
+  - `:status`
+  - `grpc-message`
+  - `grpc-status`
+  - `armeria.grpc.ThrowableProto-bin`
+
+## 🏚️ Deprecations
+
+- <type://AbstractUnwrappable#delegate()> has been deprecated in favor of <type://Unwrappable#unwrap()>.
+
+## ☢️ Breaking changes
+
+- You can specify an absolute URL *only when* a <type://WebClient> was created *without* a base URL.
+  It was previously allowed to specify an absolute URL even when created with a base URL. #2757
+- The method signature of
+  <type://RequestConverterFunction#convertRequest(ServiceRequestContext,AggregatedHttpRequest,Class,ParameterizedType)>
+  has been changed to support parameterized types. #2779
+- The default <type://MeterIdPrefixFunction> returned by <type://MeterIdPrefixFunction#ofDefault(String)>
+  now generates the meter IDs with different tags. #2780
+  - The `route` tag has been replaced with the `service` tag.
+- `RetrofitMeterIdPrefixFunctionBuilder` has been removed and superseded by
+  <type://RequestOnlyLog#serviceName()>. #2780
+- ZooKeeper-based service discovery: #2749 #2791
+  - You now must specify <type://ZooKeeperRegistrationSpec> or <type://ZooKeeperDiscoverySpec> when creating
+    <type://ZooKeeperUpdatingListener> or <type://ZooKeeperEndpointGroup>.
+  - `NodeValueCodec` has been removed.
+  - `ZooKeeperEndpointGroupBuilder.codec()` and `ZooKeeperUpdatingListenerBuilder.codec()` have been removed.
+- `ByteBufHttpData` has been replaced with <type://PooledHttpData>. #2448
+- `SubscriptionOption.WITH_POOLED_OBJECTS` has been replaced with <type://PooledHttpRequest> and
```

Comment:
I guess `aggregateWithPooledObjects` is the most commonly used so might be worth mentioning it.

Reply:
Good point. Updated.